pkgs/development/libraries/fpattern/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, lib, fetchFromGitHub }:
+
+stdenv.mkDerivation (finalAttrs: {
+  version = "1.9";
+  pname = "fpattern";
+  outputs = [ "out" "dev" ];
```

Comment:
I imagine we don't really need a separate output here

Reply:
I tried to have `dev` as the only output but kept hitting issues around missing `outputBin` `outputLib` and `outputDoc`. Perhaps you meant just `outputs = [ "out" ];` ? Most of the other derivations in `/libraries/` seemed to include `dev` so I left it in. Happy to remove if you think it's necessary.

```
pkgs/development/libraries on  master [!] 
grep -R 'outputs = \[ "out' . | wc -l
568

pkgs/development/libraries on  master [!] 
grep -R 'outputs = \[ "out" "dev"' . | wc -l
486
```